sdk/src/main/java/co/omise/android/ui/AuthorizingPaymentActivity.kt

```diff
@@ -59,11 +60,17 @@ class AuthorizingPaymentActivity : AppCompatActivity() {
         supportActionBar?.title = uiCustomization.uiCustomization.toolbarCustomization?.headerText
             ?: getString(R.string.title_authorizing_payment)
```

Comment:
These can also be moved to a separate function and call the function from `onCreate`

```
private fun setupActionBarTitle() {
    val headerText = uiCustomization.uiCustomization.toolbarCustomization?.headerText
    supportActionBar?.title = headerText ?: getString(R.string.title_authorizing_payment)
}
```